app/assets/stylesheets/components/shared/c-card.scss

```diff
@@ -1,14 +1,18 @@
 
 .c-card {
+  display: flex;
   position: relative;
-  margin: 0 0 20px;
+  flex-direction: column;
+  height: $card-height;
```

Comment:
Yes, they want _all_ cards with the same height, not just by row. That's a real limitation for us and the client has to be careful about the content despite having the character limitation implemented.